src/main/java/org/tailormap/api/scheduling/TaskManagerService.java

```diff
@@ -119,6 +119,7 @@ public void updateTask(JobKey jobKey, TMJobDataMap newJobData) throws SchedulerE
                         .withMisfireHandlingInstructionFireAndProceed())
                 .build();
 
+        scheduler.addJob(jobDetail, true, true);
```

Comment:
I think this should be 

```suggestion
        scheduler.addJob(jobDetail, true);
```

As all/most of our jobs will be persisted "durable", see https://www.quartz-scheduler.org/api/2.3.0/org/quartz/Scheduler.html#addJob-org.quartz.JobDetail-boolean-boolean-